src/hyp3_isce2/insar_tops_multi_burst.py

```diff
@@ -179,3 +186,7 @@ def main():
 
     if args.bucket:
         packaging.upload_product_to_s3(product_dir, output_zip, args.bucket, args.bucket_prefix)
+
+    if args.publish_bucket:
+        prefix, s3_name = get_multiburst_prefix(output_zip)
```

Comment:
```suggestion
        publish_name = get_publish_name(output_zip)
        if args.publish_prefix is None:
            publish_prefix = get_publish_prefix(output_zip)
        
```
you could add a `--publish-prefix` argument that would allow you to customize the prefix for non-volcsarvatory use. We will possibly need to add this to ITS_LIVE at some point as well.